specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2022-08-01-preview/sqlvm.json

```diff
@@ -0,0 +1,2542 @@
+              "Friday",
+              "Saturday",
+              "Sunday"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "AutoBackupDaysOfWeek",
+              "modelAsString": true
+            }
+          }
+        },
+        "fullBackupStartTime": {
+          "format": "int32",
+          "description": "Start time of a given day during which full backups can take place. 0-23 hours.",
+          "type": "integer"
+        },
+        "fullBackupWindowHours": {
+          "format": "int32",
+          "description": "Duration of the time window of a given day during which full backups can take place. 1-23 hours.",
+          "type": "integer"
+        },
+        "logBackupFrequency": {
+          "format": "int32",
+          "description": "Frequency of log backups. 5-60 minutes.",
+          "type": "integer"
+        }
+      }
+    },
+    "KeyVaultCredentialSettings": {
+      "description": "Configure your SQL virtual machine to be able to connect to the Azure Key Vault service.",
+      "type": "object",
+      "properties": {
+        "enable": {
+          "description": "Enable or disable key vault credential setting.",
+          "type": "boolean"
+        },
+        "credentialName": {
+          "description": "Credential name.",
+          "type": "string"
+        },
+        "azureKeyVaultUrl": {
+          "description": "Azure Key Vault url.",
+          "type": "string"
+        },
+        "servicePrincipalName": {
+          "description": "Service principal name to access key vault.",
+          "type": "string"
+        },
+        "servicePrincipalSecret": {
+          "description": "Service principal name secret to access key vault.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        }
+      }
+    },
+    "ServerConfigurationsManagementSettings": {
+      "description": "Set the connectivity, storage and workload settings.",
+      "type": "object",
+      "properties": {
+        "sqlConnectivityUpdateSettings": {
+          "$ref": "#/definitions/SqlConnectivityUpdateSettings",
+          "description": "SQL connectivity type settings."
+        },
+        "sqlWorkloadTypeUpdateSettings": {
+          "$ref": "#/definitions/SqlWorkloadTypeUpdateSettings",
+          "description": "SQL workload type settings."
+        },
+        "sqlStorageUpdateSettings": {
+          "$ref": "#/definitions/SqlStorageUpdateSettings",
+          "description": "SQL storage update settings."
+        },
+        "additionalFeaturesServerConfigurations": {
+          "$ref": "#/definitions/AdditionalFeaturesServerConfigurations",
+          "description": "Additional SQL feature settings."
+        },
+        "sqlInstanceSettings": {
+          "$ref": "#/definitions/SQLInstanceSettings",
+          "description": "SQL Instance settings."
+        },
+        "azureADAuthenticationSettings": {
```

Comment:
seems like this is violating the camel case lint rule (DefinitionsPropertiesNamesCamelCase).
We would need to update it as "azureAdAuthenticationSettings" in the RP code as well

Reply:
followed your suggestion to add azureADAuthenticationSettings to custom-words.txt

Reply:
@akning-ms Could you give us suggestion on how to fix the LintDiff DefinitionsPropertiesNamesCamelCase error? Since azureAD is the name of a service, we decied to add it to the custom-words list but the validations are still throwing the error.